lib/scrape.js

```diff
@@ -91,6 +122,35 @@ async function findOrganization({
   return null
 }
 
+async function findGitHubUser({ display_name }, org) {
+  if (!org) return
+
+  const displayNamePattern = /^[a-zA-Z1-9-]{1,39}$/
+
+  const displayNameMatches = displayNamePattern.exec(display_name)
+  if (!displayNameMatches) return
+
+  const user = await getGitHubUser(display_name)
+  if (!user) return
+
+  const login = user.login
+
+  const { competition_open_starts } = await fetchProgram()
+
+  const updatedTime = new Date(user.updated_at)
+  const openTime = new Date(competition_open_starts)
+
+  if (updatedTime.getTime() - openTime.getTime() < 0) return
+
+  const nov = await getGitHubUserCommits(login, '2017-11-28', '2017-11-30')
+  const dec = await getGitHubUserCommits(login, '2017-12-01', '2017-12-31')
+  const jan = await getGitHubUserCommits(login, '2018-01-01', '2018-01-17')
+  const orgs = nov.concat(dec).concat(jan).map(repo => repo.split('/')[0])
```

Comment:
use array spread operator, I think node 8 supports it.

Reply:
Also why not directly do `find`

Reply:
Mainly just in case someone commits to the `random_org/coalas_are_cool` repository. Very unlikely to happen, but using the organization is a more clean approach IMO.

Reply:
I mean replacing map with find,
you could do split and string match there just fine :P

But whatever. This works fine.